src/test/shamap/common.h

```diff
@@ -25,6 +25,7 @@
 #include <ripple/nodestore/DummyScheduler.h>
 #include <ripple/nodestore/Manager.h>
 #include <ripple/shamap/Family.h>
+#include <memory>
```

Comment:
I don't think this `#include` is necessary.

Reply:
removed